community/contributing/docs/reviewing-prs.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: single
+title: Reviewing Pull Requests
+sidebar:
+  nav: community
+---
+
+{% include toc %}
+
+# Review a pull request
+
+Anyone can review a documentation pull request. Visit the [pull requests](https://github.com/spinnaker/spinnaker.github.io/pulls) section in the Spinnaker website repository to see a list of all currently open pull requests.
```

Comment:
It's a good idea to use the same capitalization for UI elements that the UI uses...  `Pull requests` instead of `pull requests` and also use bold to denote UI elements ([style guide](https://www.spinnaker.io/community/contributing/docs/docs-style-guide/#use-bold-for-user-interface-elements))

`Visit the [**Pull requests**](https://github.com/spinnaker/spinnaker.github.io/pulls) section in the Spinnaker website repository to see a list of all currently open pull requests.`